lib/menu.rb

```diff
@@ -0,0 +1,32 @@
+class Menu
+  attr_reader :menu
+  attr_accessor :ordered_dishes
+
+  def initialize
+    @menu = [
+      { dish: 'pasta', price: 10.00, category: 'main_nonveg_meal' },
+      { dish: 'chickenwings', price: 20.00, category: 'main_nonveg_meal' },
+      { dish: 'fries', price: 2.00, category: 'main_nonveg_meal' },
+      { dish: 'beefburger', price: 5.00, category: 'main_nonveg_meal' }
+    ]
+  end
+
```

Comment:
What do you think about this as an alternative way to print menu items?  Is it clearer?  Does it still do what you want it to do?

```
@menu.each do |item|
  print "#{item[:dish]}, #{item[:price]}\n"
end
```